General.ruleset

```diff
@@ -0,0 +1,125 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I get this when opening the file in VS:
![image](https://user-images.githubusercontent.com/1976647/89107968-29ed6380-d435-11ea-8b4f-9da3670c05e6.png)
Would you recommend moving these to the .editorconfig? Would they be executed during build still?

Reply:
I would not do that. This way you will lose an ability to manage ruleset visually with the VS editor, which is useful, as for me.

Reply:
Fair enough.